CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [1.7.2] (July 04, 2024)
+### Feat:
+- `FormInput` of `FormMesage` now supports two new `MessageFormItem.style.layout` types: `chip` and `textarea`
+- Added `window.sbWidget` interface to allowing control of the widget state and cache clearing from a non-React environment
+
+### Update:
+- `MessageFeedbackModal` is now being displayed within the widget window
+- `autoOpen` is now being ignored in mobile view
```

Comment:
Can we mention this in the README as well? 

There's `autoOpen` prop in https://github.com/sendbird/chat-ai-widget?tab=readme-ov-file#available-props.

Reply:
done!